### PR TITLE
Fix EP0 in dynamic bridge app

### PR DESCRIPTION
**Pull request by ghost**
Currently this conflicts with EP0 Descriptor clusters, we register an override that can't provide EP0 values and it vectors into that causing commissioning failures

### CODE REVIEW

**Comment by mrjerryjohns** [comment]
Since this is really just about the descriptor cluster, we shouldn't really be over-riding the existing impl in the SDK, since that 'just works' as long as the right cluster metdata tables are provided at endpoint registration time.

Talked with @achaulk-goog, he's going to fix up the PR to ensure the descriptor cluster is present in all dynamic endpoints at endpoint registration time in the 'metadata tables', but stop short of actually registering an attribute access interface override.

File: examples/dynamic-bridge-app/linux/main.cpp [hunk 1]
```diff
@@ -294,8 +294,12 @@ int main(int argc, char * argv[])
     clusterAccess.reserve(std::extent<decltype(clusters::kKnownClusters)>::value);
     for (auto & entry : clusters::kKnownClusters)
     {
-        clusterAccess.emplace_back(chip::Optional<EndpointId>(), entry.id);
-        registerAttributeAccessOverride(&clusterAccess.back());
+        // Desciptor clusters should not be overridden.
+        if (entry.id != 29)
```
Comment (mrjerryjohns) [inline examples/dynamic-bridge-app/linux/main.cpp:298]: Please, no hard-coded numbers.

`zzz-generated/app-common/app-common/zap-generated/ids/Clusters.h` has `chip::app::Clusters::Descriptor::Id` that can be used here.

**Review by andy31415 — APPROVED** [review]

File: examples/dynamic-bridge-app/linux/DynamicDevice.cpp [hunk 2]
```diff
@@ -17,12 +17,27 @@
  *    limitations under the License.
  */
 
+#include <app-common/zap-generated/af-structs.h>
+#include <app-common/zap-generated/attribute-id.h>
+#include <app-common/zap-generated/cluster-id.h>
```
Comment (mrjerryjohns) [inline examples/dynamic-bridge-app/linux/DynamicDevice.cpp:22]: This is just right. We should not be including code-generated files from common library code.

File: examples/dynamic-bridge-app/linux/DynamicDevice.cpp [hunk 3]
```diff
@@ -17,12 +17,27 @@
...
 
+namespace {
+static const int kDescriptorAttributeArraySize = 254;
+
+DECLARE_DYNAMIC_ATTRIBUTE_LIST_BEGIN(descriptorAttrs)
+DECLARE_DYNAMIC_ATTRIBUTE(ZCL_DEVICE_LIST_ATTRIBUTE_ID, ARRAY, kDescriptorAttributeArraySize, 0),     /* device list */
+    DECLARE_DYNAMIC_ATTRIBUTE(ZCL_SERVER_LIST_ATTRIBUTE_ID, ARRAY, kDescriptorAttributeArraySize, 0), /* server list */
+    DECLARE_DYNAMIC_ATTRIBUTE(ZCL_CLIENT_LIST_ATTRIBUTE_ID, ARRAY, kDescriptorAttributeArraySize, 0), /* client list */
+    DECLARE_DYNAMIC_ATTRIBUTE(ZCL_PARTS_LIST_ATTRIBUTE_ID, ARRAY, kDescriptorAttributeArraySize, 0),  /* parts list */
+    DECLARE_DYNAMIC_ATTRIBUTE_LIST_END();
```
Comment (mrjerryjohns) [inline examples/dynamic-bridge-app/linux/DynamicDevice.cpp:38]: There must be a better solution that hand-crafting these.

**Pull request merged**